python/instrumentation/openinference-instrumentation-langchain/examples/chain_metadata.py

```diff
@@ -10,7 +10,7 @@
 endpoint = "http://127.0.0.1:6006/v1/traces"
 tracer_provider = trace_sdk.TracerProvider()
 trace_api.set_tracer_provider(tracer_provider)
-tracer_provider.add_span_processor(SimpleSpanProcessor(OTLPSpanExporter(endpoint=endpoint)))
+tracer_provider.add_span_processor(SimpleSpanProcessor(OTLPSpanExporter(endpoint)))
```

Comment:
nit: keyword is a bit safer

Reply:
That is true, but since these are just examples, I wanted to make them as least verbose as possible. On the other hand, the chance that OTEL sdk would make a breaking change here is pretty minimal